lib/src/widget_extension.dart

```diff
@@ -41,4 +54,14 @@ extension WidgetExtension on Widget {
 
   @Deprecated('Use ext.sliver instead')
   Widget get sliver => SliverToBoxAdapter(child: this);
+
+  Widget checkNullableWidget({
+    required Widget nullWidget,
+    List<dynamic>? list,
+  }) {
+    if (list.ext.isNullOrEmpty) {
+      return nullWidget;
+    }
+    return this;
```

Comment:
nullWidget is not common usage.
dynamic list is not clear usage again.

i think this method need to clearfy about why we need that.